packages/webapp/src/containers/ErrorHandler/Forbidden/Forbidden.jsx

```diff
@@ -4,15 +4,18 @@ import { useDispatch } from 'react-redux';
 import { handle403 } from '../saga';
 
 export default function Forbidden({ history }) {
- const dispatch = useDispatch();
- useEffect(() => {
-  if (history?.location?.state?.error?.response?.status === 403) {
-   dispatch(handle403());
-  } else if (history.length) {
-   history.back();
-  } else {
-   history.replace('/');
-  }
- }, []);
- return <Spinner />;
+  const dispatch = useDispatch();
+  useEffect(() => {
+    if (
+      history?.location?.state?.error?.response?.status === 403 ||
+      history?.location?.state?.error?.message?.includes('403')
+    ) {
+      dispatch(handle403());
```

Comment:
![image](https://github.com/LiteFarmOrg/LiteFarm/assets/16997894/4c10eb54-e041-45f8-840b-66c2288c0644)
I think handle403 need to check userFarm status === 'Active'
This component should probably be re-evaluated. Failure case shouldn't have caused infinite loops

Reply:
I agree, the component itself never renders because going into this state would get you logged out. Which would also get you to the login page. 
Do you think it's better if we do a bigger review of this before merging?

Reply:
Does this work?
That Forbidden component was a horrible mistake :1st_place_medal: 
https://github.com/LiteFarmOrg/LiteFarm/pull/2790
https://lite-farm-7lsmxoqps-jimmyrowland.vercel.app/

Reply:
I completely missed this @JimmyRowland. This works fine!